docs/Home.md

```diff
@@ -1,3 +1,48 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+This repository generates a new project based on our preferences, by running a simple script. 
+
+Example: `./newproject.sh -t crt -p co.myproject.example -n "MyProjectExampleCoroutine"`
+
+This script must include all essentials by default, while optional features can be appended with flags. To prevent this repository from becoming a dumping playground, features can be rejected too.
+
+Planning to contribute? Let's have a look at the criteria:
+
+How do we decide if a feature is **essential**? 👍
+
+- It is always implemented in the projects we've worked on.
+- It allows developers to avoid boilerplate setup, maintain the code quality and have good experience for themselves.
+- It is a Google recommended component.
+
+How do we decide if a feature is **optional**? 🚩
+
+- It is regularly implemented in the project's we've worked on.
+- It is an alternative to a newer dominant option.
+
+How do we decide if a feature is **rejected**? 👎
+
+- It is barely implemented in the project's we've worked on due to being a specific project requirement.
+- It has been replaced completely by a newer dominant option.
+
+Let's have a look at some examples 🔎
+
+**Essential**:
+- [Timber](https://github.com/JakeWharton/timber): A logging library which is always used in all the projects we've worked on.
+- [Firebase App Distribution](https://firebase.google.com/docs/app-distribution): Continuous delivery always requires a boilerplate setup.
+- [Detekt](https://github.com/detekt/detekt): Code smell analysis is important to maintain code quality.
+- [Hilt](https://developer.android.com/training/dependency-injection/hilt-android): Dependency injection provides a good developer experience.
+- [Navigation Component](https://developer.android.com/guide/navigation/navigation-getting-started): A suite of libraries for in-app navigation which is recommended by Google.
+
+**Optional**:
+- [Lottie](https://github.com/airbnb/lottie-android): Loading animations with a JSON file is regularly used, but not every project needs it.
```

Comment:
This could be considered as `Rejected`, the same as `Skeleton Layout` too IMO. Instead of mentioning `Lottie` as `Optional`, I think we can list `Glide - an image loader` here because almost project needs an image loader, but not always `Glide` 🤓

Reply:
>I think we can list Glide - an image loader here because almost project needs an image loader, but not always Glide 

@luongvo do you mean

>....but not always `Lottie`
?

because 10/10 recent projects we had we always use Glide 🤔

Reply:
@sleepylee Okay if 10/10 recent projects we had we always use Glide so Glide is an essential feature, I may be wrong on this part.

On another side, I still don't think we can consider `Lottie` as Optional, it should be the same as `Skeleton Layout` in example.

Reply:
@luongvo At this moment, I think I have used `Lottie` in at least of half of my projects, while for `Skeleton` it's zero.

Anyways, I'm totally open to finding a more clear example. Do you have any suggestion? 🙇

Reply:
@luongvo yep probably we can reject it too as Lottie's setup is relatively simple with a library import. Possibly for those cases, depending on the project's need - developers should setup themselves instead of us maintaining it.

Reply:
@Tuubz I think we have Skeleton loading view in The 1 + DeeMoney (not sure, maybe Toyota Wallet also) 🤔

Reply:
I'll revise the rules between "optional" & "rejected" a bit more, so it's more clear and strict + adapt the examples accordingly 👍

Reply:
@Tuubz @sleepylee How about listing [RxPermissions](https://github.com/tbruyelle/RxPermissions) as an Optional feature here instead? I see most of our projects use it and it's quite convenience for our permission handling steps too 🤔

Reply:
@luongvo that works 👍